Nodes/common.mk

```diff
@@ -0,0 +1,3 @@
+EXTERNAL_MODULE_DIRS += $(RIOTBASE)/../modules
```

Comment:
Using `$(RIOTBASE)` to reference the external modules, which are (in this project) always in `Nodes/modules/` is imo not the right approach. My reasoning is, that in order for this to work, the `RIOTBASE` must always point towards `Nodes/RIOT`.  This breaks some setups where you might want to use a RIOT setup somewhere else. E.g., my setup breaks, as I have a special RIOT folder where I test out my fixes to get it compile on macOS. (Obviously, I could still symlink or clone my repository into `Nodes/RIOT` as a workaround)

I would propose to change it into `EXTERNAL_MODULE_DIRS += ../modules/`. What do you think?

Reply:
I see. I just pushed another proposal. The entry-point for applications is now wherever `common.mk` is located. That is used as reference (`ROOT_DIR`) for `modules` and `RIOT` instead. Could you give it a try with your modified path?